### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/allstar/actions/runs/3654427931#summary-9984878732

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/allstar/commit/eafc8442ad81e133fe9a8bb4c17e8fa116eac99a

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

### CODE REVIEW

File: pkg/policies/action/test_workflows/gradle-wrapper-validate-outdated.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ name: "GW Validate Workflow"
 on: [push, pull_request]
 
 jobs:
-  gw_validat3:
+  gw_validate:
```
Comment (jsoref) [inline pkg/policies/action/test_workflows/gradle-wrapper-validate-outdated.yaml:5]: I'm puzzled by this...

**Review by jsoref — COMMENTED** [review]
Most corrections were automatically suggested by Google Sheets. All fault mine

**Comment by coheigea** [comment]
LGTM but can you rebase the PR?

**Comment by jsoref** [comment]
@coheigea: done

**Comment by coheigea** [comment]
@jeffmendoza It could be included in the release as well?

**Review by jeffmendoza — APPROVED** [review]

**Pull request merged**